src/minimega/dnsmasq.go

```diff
@@ -190,6 +444,9 @@ func dnsmasqStart(ip, min, max, hosts string) error {
 			fmt.Sprintf("--pid-file=%v/dnsmasq.pid", d.Path),
 			"-o",
 			"-k",
+			fmt.Sprintf("--hostsdir=%v", d.Hostdir),
```

Comment:
Hm, looks like these aren't available in `dnsmasq` version `2.72`, we should add a min version check in `external.go`.